andela/AndelaEatsUI
# #160476786 View orders on admin dashboard (Functionality)

#### What does this PR do?
- Adds functionality for viewing submitted/requested meal orders on the admin dashboard page
#### Description of Task to be completed?
- Fetch and replace static contents with dynamic contents
#### How should this be manually tested?
- Log in to the app
- Switch to the admin dashboard from the user profile drop-down menu or [here](http://andelaeats-dev.andela.com:3000/admin/orders)
- You should see orders displayed
#### What are the relevant pivotal tracker stories?
- [ #160476786](https://www.pivotaltracker.com/story/show/160476786)
#### Background Context
The meal orders currently being fetched are resourced using the apiary mock server for now until the backend functionality for the orders endpoint is made available
#### Screenshots
![image](https://user-images.githubusercontent.com/26303683/45492939-76f2be80-b765-11e8-9a23-dd80353b8d42.png)
